### DISCUSSION TITLE
Looking for an Elsa Workflow Expert

### DISCUSSION THREAD

**Discussion by theafricaninternetgroup** [elsa-workflows/elsa-core#3821] [General]
Previously, I shared my concern in issue #3585 regarding my project and the challenge I am facing in incorporating Elsa for designing a UI that is driven by Elsa workflow. Despite my attempts to comprehend Elsa, I am encountering obstacles. Thus, I am seeking assistance from anyone within this forum who is willing to help me. I have also posted a job on Upwork (https://www.upwork.com/ab/applicants/1638491477528940544/job-details), but I understand that the chances of finding an expert are minimal. Therefore, I am appealing to anyone kind enough to lend their expertise and aid me in this matter.

**Comment by theafricaninternetgroup** [comment]
Kindly note I am willing to pay for the help anyone who will be kind enough to help

**Comment by theafricaninternetgroup** [comment]
@sfmskywalker Have you not setup your consultancy.I know you are busy working on v3,but I really need help.Our project has missed delivery deadline because of this.

**Reply by sfmskywalker** [reply]
Hi @theafricaninternetgroup , I am sorry to hear about the difficulties of finding help. I am working on building a team of experts. If you send me an email at sipke@skywalker-digital.com we can discuss this in further detail.

**Reply by theafricaninternetgroup** [reply]
@sfmskywalker I have sent the email.Hoping to hear from you soon.
